src/app/helper/files.util.ts

```diff
@@ -0,0 +1,8 @@
+export class FilesUtil {
+  public static saveText(text: string, filename: string): void {
+    var a = document.createElement('a')
```

Comment:
`var` should be a `const` instead, and maybe `a` could be more descriptive although i understand you linking it to it's element.

Reply:
A type could be added as well :-)